app/lib/encryption/room.ts

```diff
@@ -19,9 +21,19 @@ import {
 	utf8ToBuffer
 } from './utils';
 import { Encryption } from './index';
+import { IUser } from '../../definitions/IUser';
 
 export default class EncryptionRoom {
-	constructor(roomId, userId) {
+	ready: boolean;
+	roomId: string;
+	userId: string;
+	establishing: boolean;
+	readyPromise: Deferred<unknown>;
+	sessionKeyExportedString!: string | ByteBuffer;
+	keyID!: string;
```

Comment:
You are using Non-null assertion because it was not declared, wouldn't it be better to declare it?